src/interfaces/plug_keyring.ts

```diff
@@ -1,7 +1,16 @@
 import type PlugWallet from '../PlugWallet'
+import {
+    JSONWallet
+  } from '../interfaces/plug_wallet';
 
 export interface PlugState {
-    wallets: Array<PlugWallet>;
+    wallets: Array<JSONWallet>;
     password?: string;
     currentWalletId?: number;
+}
+
+export interface PlugStateInstance {
```

Comment:
@rocky-fleek what do u think here? We need to separate the interface what we gonna save on the storage and what have the state. I think we could use `Array<PlugWallet> | Array<JSONWallet>` Or `Array<PlugWallet|JSONWallet>` either